data/data-pipeline/data_pipeline/etl/score/etl_score.py

```diff
@@ -1,91 +1,19 @@
-import collections
 import functools
-from pathlib import Path
 import pandas as pd
 
 from data_pipeline.etl.base import ExtractTransformLoad
+from data_pipeline.score.score_calculator import ScoreCalculator
+import data_pipeline.score.field_names as FN
+
 from data_pipeline.utils import get_module_logger
-from data_pipeline.etl.score.score_calculator import ScoreCalculator
+from . import constants
```

Comment:
@esfoobar-usds this is what we're doing in `etl_score_post.py`. It's quite a lot of constants; I'm concerned that if we listed out all of them it kind of defeats the purpose of having a constants file (because it puts the constant names in multiple places that then have to be maintained, and then it takes up lots of redundant space). 

I could refactor this and the identical line in `etl_score_post.py` to ue the pattern I use elsewhere for the field names set of constants (`import data_pipeline.score.field_names as FN`) so that this line becomes `import data_pipeline.etl.constants as CONSTANTS`. Thoughts?

Reply:
I see -- even though we don't use `score_post` on Docker -- I would need to theckout your branch and run a Docker instance from it to see if get any issues.

The key line is this, we install the `data_pipeline` as a Poetry package, and thus the relative imports don't work -- at least that's what Nat and I saw at the beginning.

Did you mean `import data_pipeline.etl.score.constants as CONSTANTS`?

Reply:
> Did you mean import data_pipeline.etl.score.constants as CONSTANTS? 

Oh, yes

Okay! Want to run this on Docker and see what happens? I can also do this tomorrow.

Reply:
Any reason we wouldn't be fine with just `import data_pipeline.etl.score.constants`, and then when we need to reference them, using `constants.FIELD_NAME`?

Reply:
im fine with that!

Reply:
changed this!